src/libcore/ops/range.rs

```diff
@@ -320,15 +335,22 @@ impl<Idx: PartialOrd<Idx>> RangeInclusive<Idx> {
 /// }
 /// ```
```

Comment:
Ditto my comment on the similar block in `RangeTo`, about putting the compile error into the sample.